src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents an association between a Person and a Tag in the address book.
```

Comment:
Good but can be more specific. 
e.g. `Represents an adding or deleting a Tag to a Person.`